src/spoof.ts

```diff
@@ -293,9 +291,12 @@ const getElementBox = async (
   }
 }
 
-export function path (point: Vector, target: Vector, options?: number | PathOptions): Vector[] | TimedVector[]
-export function path (point: Vector, target: BoundingBox, options?: number | PathOptions): Vector[] | TimedVector[]
-export function path (start: Vector, end: BoundingBox | Vector, options?: number | PathOptions): Vector[] | TimedVector[] {
+/** Generates a set of points for mouse movement between two coordinates. */
+export function path (
```

Comment:
Overloads not necessary here since all 3 of them had very close to the same args. Just used a union.